### PR TITLE
feature: S3C-3525 Create Tenant-2

### PR DESCRIPTION
**Pull request by saitharunthodupunuru**
* Adding necessary Scality specific implementation classes by extending or replacing vmware classes
* Modifying application.properties using original vmware suggested properties for Vault and App info

### CODE REVIEW

File: osis-core/src/main/java/com/vmware/osis/scality/AppEnv.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache License 2.0
  */
 
-package com.vmware.osis.platform;
+package com.vmware.osis.scality;
```
Comment (NicolasT) [inline osis-core/src/main/java/com/vmware/osis/scality/AppEnv.java:6]: Who are we to define a `com.vmware.osis.scality` package whilst we don't own the `(osis.)vmware.com` domain?

Comment (saitharunthodupunuru) [reply osis-core/src/main/java/com/vmware/osis/scality/AppEnv.java]: @NicolasT 
Thank you. Updated all packages from `com.vmware.*` to `com.scality.*`

**Review by anurag4DSB — APPROVED** [review]

**Review by ilkescality — APPROVED** [review]

File: osis-app/src/main/java/com/scality/osis/resource/OsisController.java [hunk 2]
```diff
@@ -1,15 +1,11 @@
-/**
```
Comment (NicolasT) [inline osis-app/src/main/java/com/scality/osis/resource/OsisController.java:1]: You're definitely breaching the original license by removing copyright headers etc.

**Review by NicolasT — CHANGES_REQUESTED** [review]

File: osis-app/src/main/java/com/scality/osis/resource/OsisController.java (hunk 2 above)
Comment (saitharunthodupunuru) [reply osis-app/src/main/java/com/scality/osis/resource/OsisController.java:1]: Fixed this and added copyright headers along with `Scality`.

**Comment by NicolasT** [comment]
As discussed on Slack, my .02 how this could be structured:
- Upstream stubs are imported as-is, as was done, in the VMWare namespace
- Within the same JAR, whenever we need a new package/class file, we create it as a package within the `com.scality` domain
- Whenever we need to make *changes to existing stub files* (we can't work around by inheritance, implementing an interface,...) we copy the file from the VMWare namespace to a Scality-owned one (package name), and add a Scality copyright header (next to the VMWare one, not replacing it!)

This way, we don't claim (pure) VMWare code is Scality's, and we don't pretend Scality (original or modified stubs) code is VMWare's, neither of which we should do (cfr. the Apache-2 license).

**Comment by saitharunthodupunuru** [comment]
> As discussed on Slack, my .02 how this could be structured:
> 
> * Upstream stubs are imported as-is, as was done, in the VMWare namespace
> * Within the same JAR, whenever we need a new package/class file, we create it as a package within the `com.scality` domain
> * Whenever we need to make _changes to existing stub files_ (we can't work around by inheritance, implementing an interface,...) we copy the file from the VMWare namespace to a Scality-owned one (package name), and add a Scality copyright header (next to the VMWare one, not replacing it!)
> 
> This way, we don't claim (pure) VMWare code is Scality's, and we don't pretend Scality (original or modified stubs) code is VMWare's, neither of which we should do (cfr. the Apache-2 license).

@NicolasT 
Updated the PR with the suggested changes. Please review

**Comment by saitharunthodupunuru** [comment]
@NicolasT Can you please do a re review and approve this?

**Review by NicolasT — APPROVED** [review]
(Not really reviewed, assuming comments were applied).

**Review by anurag4DSB — APPROVED** [review]

**Pull request merged**